### PR TITLE
Promote pid limits to ga

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Follow up from https://github.com/kubernetes/kubernetes/pull/94140/

(used https://github.com/kubernetes/website/pull/23332/ as an example)

### CODE REVIEW

**Comment by kbhawkey** [comment]
Hi @SergeyKanzhelev .
It looks like these changes are meant for the v1.20 release.
You should create your changes on the `dev-1.20` branch instead of the `master` branch.
Thanks!
/cc @annajung

**Comment by SergeyKanzhelev** [comment]
> Hi @SergeyKanzhelev .
> It looks like these changes are meant for the v1.20 release.
> You should create your changes on the `dev-1.20` branch instead of the `master` branch.
> Thanks!
> /cc @annajung

sorry, fixed.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -290,6 +286,12 @@ different Kubernetes components.
 | `SupportIPVSProxyMode` | `false` | Beta | 1.9 | 1.9 |
 | `SupportIPVSProxyMode` | `true` | Beta | 1.10 | 1.10 |
 | `SupportIPVSProxyMode` | `true` | GA | 1.11 | - |
+| `SupportNodePidsLimit` | `false` | Alpha | 1.14 | 1.14 |
+| `SupportNodePidsLimit` | `true` | Beta | 1.15 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:290]:
```suggestion
| `SupportNodePidsLimit` | `true` | Beta | 1.15 | 1.19 |
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -290,6 +286,12 @@ different Kubernetes components.
...
+| `SupportNodePidsLimit` | `true` | GA | 1.20 | - |
+| `SupportPodPidsLimit` | `false` | Alpha | 1.10 | 1.13 |
+| `SupportPodPidsLimit` | `true` | Beta | 1.14 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:293]:
```suggestion
| `SupportPodPidsLimit` | `true` | Beta | 1.14 | 1.19 |
```

**Comment by tengqm** [comment]
/lgtm
Thanks.

**Comment by annajung** [comment]
Hello, 1.20 docs lead here. Can we also get a lgtm from the sig? @kubernetes/sig-node-pr-reviews

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-23883--kubernetes-io-vnext-staging.netlify.app/docs/reference/command-line-tools-reference/feature-gates/

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -290,6 +286,12 @@ different Kubernetes components.
 | `SupportIPVSProxyMode` | `false` | Beta | 1.9 | 1.9 |
 | `SupportIPVSProxyMode` | `true` | Beta | 1.10 | 1.10 |
 | `SupportIPVSProxyMode` | `true` | GA | 1.11 | - |
+| `SupportNodePidsLimit` | `false` | Alpha | 1.14 | 1.14 |
+| `SupportNodePidsLimit` | `true` | Beta | 1.15 | 1.19 |
+| `SupportNodePidsLimit` | `true` | GA | 1.20 | - |
```
Comment (kbhawkey) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:291]: @SergeyKanzhelev , would you have time to add a description for `SupportNodePidsLimit` (~ line 535) similar to:
```
SupportPodPidsLimit: Enable the support to limiting PIDs in Pods.
```

**Comment by kbhawkey** [comment]
LGTM

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 3 above)
Comment (SergeyKanzhelev) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:291]: PTAL

**Comment by kbhawkey** [comment]
Thanks @SergeyKanzhelev 
/approve

**Pull request merged**